### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by 0xDrMoe**

### CODE REVIEW

File: Resources/Lang/es_ES.json
Comment (0xDrMoe) [inline Resources/Lang/es_ES.json:1]: this needs some fixing, whoever is responsible for Spanish needs to re-update the Neutrals

File: Resources/Lang/pt_BR.json
Comment (0xDrMoe) [inline Resources/Lang/pt_BR.json:1]: this needs some fixing, whoever is responsible for Portuguese needs to re-update the Neutrals

File: Resources/Lang/zh_TW.json
Comment (0xDrMoe) [inline Resources/Lang/zh_TW.json:1]: needs some updates

File: Resources/Lang/pt_BR.json
Comment (ryuk201198) [reply Resources/Lang/pt_BR.json:1]: @0xDrMoe Use crowdin/discord for translation related discussions. Not every translator looks at git PRs

**Pull request merged**